documentation/multi_namespace_support.md

```diff
@@ -1,6 +1,6 @@
 # Support for multiple namespaces
 
-The operator can import dashboards from either one, some or all namespaces. By default it will only look for dashboards in its own namespace.
+The operator can import dashboards from either one, some or all namespaces. By default it will look for dashboards in all namespaces.
 By setting the `--scan-all` or `--namespaces` flags the operator can watch for dashboards in other namespaces.
 
```

Comment:
Im thinking maybe adding or rephrasing the sentence about the scan all flag saying that by default the --scan all flag is set to true. So people dont try to use it even though its already in place. What do you think?

Reply:
Yeah, good idea

Reply:
i would say "these values" are set to true by default because --scan-all and --namespaces shouldnt be used together. What about "The default for the operator is --scan-all is set to true. If specific namespaces are needed rather then them all the --namespace flag can be used." Something like that?